be/src/service/staros_worker.cpp

```diff
@@ -378,5 +379,13 @@ void shutdown_staros_worker() {
     staros::starlet::common::ShutdownHook::shutdown();
 }
 
+// must keep each config the same
+void update_staros_starcache() {
+    if (fslib::FLAGS_use_star_cache != config::starlet_use_star_cache) {
+        fslib::FLAGS_use_star_cache = config::starlet_use_star_cache;
+        (void)fslib::star_cache_init(fslib::FLAGS_use_star_cache);
```

Comment:
what if `starlet_use_star_cache` is updated from `true` to `false`

Reply:
it's ok, it will be handled correctly in `fslib::star_cache_init`